README.md

```diff
@@ -23,7 +23,7 @@ ember install ember-frost-sort
 ```
 
 ## API
-Coming soon
+Coming soon...
```

Comment:
I think it is good to get in the habit of using the actual ellipsis character as it is more semantically correct and the visual formatting of it usually differs from three periods (the space between dots is generally less). I'm not sure if there are implications for things like screen readers or not. Not a reason to hold up the PR but it is good to get in the habit of things like this.